graylog2-server/src/main/java/org/graylog2/indexer/GIMMapping.java

```diff
@@ -35,143 +51,143 @@
-            .add("user_email")
-            .add("user_id")
-            .add("user_category")
-            .add("user_type")
-            .add("user_priority")
+            .add(UserFields.USER_COMMAND)
+            .add(UserFields.USER_COMMAND_PATH)
+            .add(UserFields.USER_DOMAIN)
+            .add(UserFields.USER_EMAIL)
+            .add(UserFields.USER_ID)
+            .add(UserFields.USER_CATEGORY)
+            .add(UserFields.USER_TYPE)
+            .add(UserFields.USER_PRIORITY)
             .add("source_user_domain")
-            .add("source_user_email")
+            .add("source_user_email") // SourceFields.SOURCE_USER_EMAIL is deprecated
             .add("source_user_id")
             .add("source_user_category")
             .add("source_user_type")
             .add("source_user_priority")
             .add("target_user_domain")
-            .add("target_user_email")
-            .add("target_user_id")
+            .add(UserFields.TARGET_USER_EMAIL)
+            .add(UserFields.TARGET_USER_ID)
             .add("target_user_category")
             .add("target_user_type")
             .add("target_user_priority")
-            .add("http_application")
-            .add("http_content_type")
-            .add("http_host")
+            .add(HttpFields.HTTP_APPLICATION)
+            .add(HttpFields.HTTP_CONTENT_TYPE)
+            .add(HttpFields.HTTP_HOST)
             .add("http_path")
-            .add("http_referrer")
-            .add("http_method")
-            .add("http_response")
-            .add("http_user_agent")
-            .add("http_user_agent_name")
-            .add("http_url")
-            .add("http_url_category")
-            .add("http_user_agent_os")
-            .add("http_version")
-            .add("http_xff")
-            .add("host_as_domain")
-            .add("host_as_isp")
-            .add("host_as_organization")
-            .add("host_as_number")
-            .add("host_geo_city")
-            .add("host_geo_country")
+            .add(HttpFields.HTTP_REFERER)
+            .add(HttpFields.HTTP_METHOD)
+            .add(HttpFields.HTTP_RESPONSE)
+            .add(HttpFields.HTTP_USER_AGENT)
+            .add(HttpFields.HTTP_USER_AGENT_NAME)
+            .add(HttpFields.HTTP_URL)
+            .add(HttpFields.HTTP_URL_CATEGORY)
+            .add(HttpFields.HTTP_USER_AGENT_OS)
+            .add(HttpFields.HTTP_VERSION)
+            .add("http_xff") // HttpFields.HTTP_XFF is "http_version"
+            .add(HostFields.HOST_AS_DOMAIN)
+            .add(HostFields.HOST_AS_ISP)
+            .add("host_as_organization") // There is no HostFields.HOST_AS_ORGANIZATION, only HostFields.HOST_AS_ORGANIZATION_NAME
+            .add(HostFields.HOST_AS_NUMBER)
+            .add("host_geo_city") // There is no HostFields.HOST_GEO_CITY, only HostFields.HOST_GEO_CITY_NAME
+            .add("host_geo_country") // There is no HostFields.HOST_GEO_COUNTRY, only HostFields.HOST_GEO_COUNTRY_NAME
```

Comment:
Will discuss these with @bud1979 and @miwent in next week's meeting

Reply:
Yes,  these have the _name after them.  So they are:  `host_geo_city_name` and `host_geo_country_name`